### PR TITLE
Fix nullable markup policy settings

### PR DESCRIPTION
**Pull request by DaniyarAskarov**

### CODE REVIEW

File: Api/Models/Markups/MarkupPolicySettings.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@ namespace HappyTravel.Edo.Api.Models.Markups
         /// <summary>
         ///     Policy description.
         /// </summary>
-        public string Description { get; }
+        public string? Description { get; } = string.Empty;
```
Comment (DFSko) [inline Api/Models/Markups/MarkupPolicySettings.cs:28]: not needed set default value for nullable types

File: Api/Models/Markups/MarkupPolicySettings.cs [hunk 2]
```diff
@@ -51,6 +51,6 @@ namespace HappyTravel.Edo.Api.Models.Markups
         /// <summary>
         ///     Destination of booking from the mapper
         /// </summary>
-        public string DestinationScopeId { get; }
+        public string? DestinationScopeId { get; } = string.Empty;
```
Comment (DFSko) [inline Api/Models/Markups/MarkupPolicySettings.cs:54]: not needed set default value for nullable types

File: Api/Models/Markups/MarkupPolicySettings.cs [hunk 3]
```diff
@@ -8,9 +8,23 @@ namespace HappyTravel.Edo.Api.Models.Markups
...
+            LocationScopeId = locationScopeId;
+            LocationScopeType = locationScopeType;
+            DestinationScopeId = destinationScopeId;
+        }
+
+
+        public MarkupPolicySettings(string? description, MarkupFunctionType functionType, decimal value,
             Currencies currency, string locationScopeId = "", SubjectMarkupScopeTypes locationScopeType = SubjectMarkupScopeTypes.NotSpecified,
-            string destinationScopeId = "")
+            string? destinationScopeId = "")
```
Comment (DFSko) [inline Api/Models/Markups/MarkupPolicySettings.cs:27]: I guess for nullable properties you can pass null value

**Review by DFSko — APPROVED** [review]

File: Api/Services/Markups/MarkupPolicyExtensions.cs [hunk 4]
```diff
@@ -13,11 +13,12 @@ public static MarkupPolicySettings GetSettings(this MarkupPolicy policy)
                 ? policy.SubjectScopeId
                 : null;
 
-            return new(description: policy.Description,
+            return new(
+                description: policy.Description,
```
Comment (AlexeyShok) [inline Api/Services/Markups/MarkupPolicyExtensions.cs:17]: Invalid formatting

**Pull request merged**